web/src/containers/ExecutiveSummary.js

```diff
@@ -1,33 +1,20 @@
 import PropTypes from 'prop-types';
 import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
-import { useHistory } from 'react-router-dom';
 
 import ExecutiveSummaryBudget from './ExecutiveSummaryBudget';
 import Waypoint from './ConnectedWaypoint';
 import Dollars from '../components/Dollars';
 import Review from '../components/Review';
 import { Section, Subsection } from '../components/Section';
 
-import { jumpTo } from '../actions/app';
 import { selectApdYears } from '../reducers/apd.selectors';
 import {
   selectBudgetExecutiveSummary,
   selectBudgetGrandTotal
 } from '../reducers/budget.selectors';
 
-const ExecutiveSummary = ({ data, jumpAction, total, years }) => {
-  const history = useHistory();
-
-  const navigateToActivity = (key, index) => e => {
-    e.stopPropagation();
-    e.preventDefault();
-
-    jumpAction(`activity-${key}-overview`);
-    history.push(`/apd/activity/${index}`);
-    window.scrollTo(0, 0);
-  };
-
+const ExecutiveSummary = ({ data, total, years }) => {
```

Comment:
With the addition of the `ScrollToWaypoint` component, the need to prevent the click event, dispatch a jump action, push to history, and scrollTo the top of the window within individual components is not needed! 🎉